Don't put localtime.c EXIVCSRC. It's already a part of the library.
--- a/src/Makefile
+++ b/src/Makefile
@@ -140,9 +140,7 @@
             utils.cpp
 
 # C source files for the Exiv2 application
-ifndef HAVE_TIMEGM
-EXIVCSRC  = localtime.c
-endif
+EXIVCSRC  =
 
 # ******************************************************************************
 # Library
